src/components/simple-billboard/index.js

```diff
@@ -0,0 +1,118 @@
+import React, { useCallback, useMemo } from 'react';
+import PropTypes from 'prop-types';
+
+import { Icon } from 'nr1';
+
+/**
+ * @param {Object} metric - metric name, value, previousCalue
+ * @param {string} prefix - prefix for metric.value (i.e. '$')
+ * @param {string} suffix - suffix for metric.value (i.e. users)
+ * @param {string} className -  SCSS class name - gets added to existing JSX classes
+ * @param {Object} style - SCSS style - gets added to JSX - overwrites existing/duplicate CSS properties
+ * @return {JSX Object} - RENDERING name, value, up/down trend when previousValue present
+ */
+const SimpleBillboard = ({ metric, prefix, suffix, className, style }) => {
+  const metricValue = useMemo(
+    () => (isNaN(metric.value) ? '-' : Number(metric.value), [metric.value])
+  );
+  const metricPreviousValue = useMemo(
+    () => (
+      isNaN(metric.previousValue) ? '-' : Number(metric.previousValue),
+      [metric.previousValue]
+    )
+  );
+
+  const MetricStatus = useCallback(
+    (difference) => {
+      let attributes = {
+        type: Icon.TYPE.INTERFACE__CARET__CARET_BOTTOM__WEIGHT_BOLD,
+        color: 'red',
+      };
+      if (difference === 0) {
+        attributes = {
+          type: Icon.TYPE.INTERFACE__CARET__CARET_RIGHT__WEIGHT_BOLD,
+          color: 'gold',
+        };
+      } else if (difference > 0) {
+        attributes = {
+          type: Icon.TYPE.INTERFACE__CARET__CARET_TOP__WEIGHT_BOLD,
+          color: 'green',
+        };
+      }
+      return (
+        <div className="metric-status">
+          <Icon {...attributes} />
+        </div>
+      );
+    },
+    [metricValue - metricPreviousValue]
+  );
+
+  const formatValue = useMemo(() => {
```

Comment:
Since this returns a value as opposed to a function, it would probably be better to name this `formattedValue`.